HelgaShiryaeva/assignment4/IslandsCounterTest

```diff
@@ -0,0 +1,105 @@
+package assignment4;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+public class IslandsCounterTest {
+
+    @Test
+    public void threeComponentsTest() {
+        boolean tiles[][] = {
+                {false, true, false, true},
+                {true, true, false, false},
+                {false, false, true, false},
+                {false, false, true, false}
+        };
+        IslandsCounter islandsCounter = new IslandsCounter();
```

Comment:
`IslandsCounter` is stateless, so it typically would be created either directly as a field in `IslandsCounterTest` or in `setUp()` method.